sunpy/util/config.py

```diff
@@ -123,3 +123,26 @@ def _get_user_configdir():
     if not _is_writable_dir(configdir):
         raise RuntimeError(f'Could not write to SUNPY_CONFIGDIR="{configdir}"')
     return configdir
+
+
+def write_config(overwrite=False):
+    """
+    Copies default config file to user config directory.
+    """
+    config_filename = 'sunpyrc'
+    module_dir = Path(sunpy.__file__).parent
+    config_file = module_dir / 'data' / config_filename
+    user_config_dir = Path(_get_user_configdir())
+    user_config_file = user_config_dir / config_filename
+
+    if not _is_writable_dir(user_config_dir):
+        raise RuntimeError(f'Could not write to SunPy config directory="{user_config_dir}"')
+
+    # No config file exists in user directory
+    if not user_config_file.exists():
+        shutil.copyfile(config_file, user_config_file)
+
+    # File exists and needs to be overwritten
+    elif overwrite:
+        os.remove(user_config_file)
+        shutil.copyfile(config_file, user_config_file)
```

Comment:
I feel like this this if/elif block could be rewritten in a different manner. 

Also I think if the file already exists, then you we should tell the user that it does and if they want to write to it, they have to use overwrite.

Reply:
@nabobalis I wanted to add an `else` after the `elif` block to display a warning message, but I was unsure if it's necessary. That's why I structured the if/elif block in this way.  Now that you mention it, I think it can be rewritten in a better way.

How should I tell the user about existing config file -- using `warnings.warn(message)` or a simple `print(message)`? I think a warning message would be better, but that will also require importing another library simply for this.

Reply:
P.S. Any idea why some tests are failing?

Reply:
The test failures seem to be unrelated.

I think a warning.warn would be a better idea.